### PR TITLE
bug fix attempts. still looking for fixes (testing put requests for adminIncidentsRouter)

### PR DESCRIPTION
**Pull request by dylan-rinella**
# Description
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Test A
- [ ] Test B

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by RyderBlues — COMMENTED** [review]
This looks good, are you sure you need your beforeAll/beforeEach statements?

**Comment by dylan-rinella** [comment]
> This looks good, are you sure you need your beforeAll/beforeEach statements?

Not really sure, I was looking at what Joe did on his end, and I liked his approach. I cant take too too much credit :p
Still working on this :)

**Comment by devbrunopaula** [comment]
@dylan-rinella Your pull request template its all blank.

File: __tests__/routes/incidents-update-byid.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+describe('initial test', () => {
+  test('app should run as expected', () => {
+    expect(true).not.toBe(false);
+  });
+});
+
+describe('main tests', () => {
+  test('update information as specified', async () => {
+    const incidentOne = await Incidents.getIncidentById(1);
+    console.log(incidentOne);
```
Comment (devbrunopaula) [inline __tests__/routes/incidents-update-byid.test.js:22]: Don't forget to remove your console logs.. I always forget them too.

**Review by devbrunopaula — CHANGES_REQUESTED** [review]
Great Clean Code.. you are the best

File: __tests__/routes/incidents-update-byid.test.js (hunk 1 above)
Comment (dylan-rinella) [reply __tests__/routes/incidents-update-byid.test.js:22]: Good lookin' out! Will fix :)

**Comment by Iamlegend-Imani** [comment]
Does this file need to be deleted?

**Pull request closed without merging**